src/AppBundle/Controller/BlogController.php

```diff
@@ -56,6 +56,14 @@ public function indexAction($page)
      */
     public function postShowAction(Post $post)
     {
+        // Symfony provides a function called 'dump()' which is an improved version
+        // of the 'var_dump()' function. It's useful to quickly debug the contents
+        // of any variable, but it's not available in the 'prod' environment to
+        // prevent any leak of sensitive information.
```

Comment:
I suggest adding a sentence in the comment explaining that this feature is provided by DebugBundle, and so can only be used in environments where the bundle is enabled.